src/js/shot-generator/Editor.js

```diff
@@ -623,10 +659,31 @@ const SceneManager = connect(
         }
       }
     }, [machineState.value, camera, cameraControlsView.current, mainViewCamera])
-
     // console.log('SceneManager render', sceneObjects)
+    
     const components = Object.values(sceneObjects).map(props => {
-        switch (props.type) {
+      
```

Comment:
Not using `types`, `sceneObjectsSorted`, `items` for anything. Can we remove this block?